pkg/apis/serving/fieldmask.go

```diff
@@ -710,10 +710,12 @@ func SecurityContextMask(ctx context.Context, in *corev1.SecurityContext) *corev
 	// SeccompProfile defaults to "unconstrained", but the safe values are
 	// "RuntimeDefault" or "Localhost" (with localhost path set)
 	out.SeccompProfile = in.SeccompProfile
-
+	// Allow setting Privileged to only false
```

Comment:
nit: align comment with the other one 
`// Only allow setting AutomountServiceAccountToken to false`